### ISSUE TITLE
Don't install duplicate firewall rules

### ISSUE THREAD

**Issue by danweller18** [osmud/osmud#53]
I am running osmud on the router and using the install firewall rule.sh and I am getting duplicates of some of the rules installed:

firewall.@rule[12]=rule
firewall.@rule[12].name='cl0-frdev'
firewall.@rule[12].target='ACCEPT'
firewall.@rule[12].proto='tcp'
firewall.@rule[12].family='ipv4'
firewall.@rule[12].src='lan'
firewall.@rule[12].src_ip='192.168.10.55'
firewall.@rule[12].dest='wan'
firewall.@rule[12].dest_ip='52.54.232.152'
firewall.@rule[12].dest_port='9543'
firewall.@rule[13]=rule
firewall.@rule[13].name='cl0-frdev'
firewall.@rule[13].target='ACCEPT'
firewall.@rule[13].proto='tcp'
firewall.@rule[13].family='ipv4'
firewall.@rule[13].src='lan'
firewall.@rule[13].src_ip='192.168.10.55'
firewall.@rule[13].dest='wan'
firewall.@rule[13].dest_ip='52.54.232.152'
firewall.@rule[13].dest_port='9543'

**Comment by kyeich** [comment]
This will be done by trying to remove all rules for an IP address before applying new rules.

**Comment by kyeich** [comment]
Additionally, there are now two new scripts - 1) commit 2) rollback. The concept of applying a new MUD file will now be considered a transaction. If all steps of the MUD file can be applied, the configuration will be committed. Otherwise all changes will be rolled-back. The install and remove firewall rule scripts have been changed to no longer commit and restart the firewall after each call. The create pending changes and at the end, the changes are committed when there are no errors encountered. The first step in applying a new MUD file is to call the remove script for the IP before starting.
